bigbluebutton-tests/playwright/parameters/parameters.spec.js

```diff
@@ -528,7 +528,7 @@ test.describe.parallel('Custom Parameters', { tag: '@ci' }, () => {
     await customParam.hideActionsBarTest();
   });
 
-  test('Override Default Locale', { tag: '@flaky-3.1' }, async ({ browser, context, page }) => {
+  test('Override Default Locale', async ({ browser, context, page }) => {
```

Comment:
flaky flag removed, but the test is failing

fix suggestion: `hasText` assertion function won't work on many selectors, as there are few "main" elements with text content. for this case, you could assert: placeholders, tooltips when hovering, sideContent titles, media-area button text, and/or settings text (would be good one check for each at leasat)